packages/openapi-generator/src/file-serializer/docs.ts

```diff
@@ -0,0 +1,71 @@
+import { EOL } from 'os';
```

Comment:
We should probably replace all occurences of `\n` with this?

Reply:
This depends I would say. In Regex the `\s` or `\n` should work on any os. However, if you build a multi line string or so you should use the `EOL` in my opinion. I would create a follow up ticket for this?